nixos/modules/services/networking/birdwatcher.nix

```diff
@@ -0,0 +1,131 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.birdwatcher;
+in
+{
+  meta.buildDocsInSandbox = false;
```

Comment:
Why?

Reply:
Becuase in the line 64 because the test suggested doing that, before some one in the nixpkgs matrix room explained to me how I can properly escape `${pkgs.bird}` with `''${pkgs.bird}`. I can probably remove it now and it should still build.